src/main/app/package.json

```diff
@@ -81,8 +81,12 @@
     }
   },
   "lint-staged": {
-    "*.{js,jsx,ts,tsx,json,css,md}": [
-      "./node_modules/.bin/prettier --write",
+    "*.{json,css,md}": [
+      "prettier --write",
+      "git add"
+    ],
+    "*.{js,jsx,ts,tsx}": [
+      "eslint --fix",
       "git add"
```

Comment:
Miksihän täällä on git add? Itse hämmentyisin jos commitin mukaan menisi jotain automaattisesti mitä itse ei haluaisi. Jos halutaan pitää niin vaihtaisin "lint-staged" nimen "lint-staged-and-git-add" tjms.

Reply:
Tässähän on siis tämä käytössä: https://github.com/okonet/lint-staged joten nimeä ei voine muuttaa. Se ajaa muuttuneille tiedostoille nuo komennot, joten mitään uusia tiedostoja se ei lisää. Tuo `git add` ei ilmeisesti enää `lint-staged` v10:stä lähtien ole pakollinen, joten taidanpa ottaa sen pois. Aiemmin se on siis tarvittu, jotta lintterin korjaukset on otettu mukaan committiin.

Reply:
Jees, katselmoin tän perjantai-iltapäivänä niin jotenkin meni silmät ristiin että tämä olisi husky komento, mutta niin siis tämähän oli tosiaan lint-staged spesifinen konffis :D

Eli kaikki ok. Hyvä jos ei tarvii, tollanen git add magia seassa hämmensi kun ei ollut ollut käytössä missään aiemmassa UI hommas.